.yarn/versions/4a91906d.yml

```diff
@@ -1,5 +1,6 @@
 releases:
-  "@radix-ui/react-tabs": patch
+  "@radix-ui/react-radio-group": patch
 
 declined:
   - primitives
+  - "@radix-ui/react-tabs"
```

Comment:
I'm confused why `tabs` is here? Did it show up in the list to set versions when you ran the command? 🤔

Reply:
On closer inspection, it looks like you have modified an existing version file. Do you know what happened?

Reply:
Not sure to be honest! I just ran the version command and it came up, so I set it to decline

Reply:
hmmm, that's really odd, we'll take a look.